rules/naming/tests/Rector/Property/MakeBoolPropertyRespectIsHasWasMethodNamingRector/Fixture/skip_method_name_after_prefix_not_matches_property_name.php.inc

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Rector\Naming\Tests\Rector\Property\MakeBoolPropertyRespectIsHasWasMethodNamingRector\Fixture;
+
+class SkipMethodNameAfterPrefixNotMatchesPropertyName
+{
+    /**
+     * @var bool
+     */
+    private $full = false;
+
+    public function isSomething()
+    {
+        return $this->full;
+    }
+
+}
+
+?>
```

Comment:
@TomasVotruba, before finishing I came up with this test fixture. Does this rule meant to rename properties in such cases? When there is nothing similar in naming but the method return the property value.

Reply:
What do you think most people expect to happen in this case?

Reply:
I wish I could know... I wouldn't ask if I knew the answer to this question :)
Maybe it is better to rename it in such cases, even though I'll need to re-write the rule to handle this case :D

Reply:
I don't know either. I'd have to ask everyone too :) I just don't know much about rule, since I haven't made it.

> Maybe it is better to rename it in such cases, even though I'll need to re-write the rule to handle this case :D

That's what I'd expect. I also try to test it on Rector to get more data, if such assumption is right. It usually helps to decide better

Reply:
Nothing special after this change in Rector itself but since we'd both expect that the rule will rename the property in such cases, I'm leaving it as it is now.